lib/logstash/devutils/rspec/spec_helper.rb

```diff
@@ -21,6 +21,12 @@
 
 Thread.abort_on_exception = true
 
+# set log4j configuration
+unless java.lang.System.getProperty("log4j.configurationFile")
```

Comment:
since `logstash-core`has been loaded above, can we decide whether or not to do this based on `LOGSTASH_VERSION`? That may more cleanly signify intent & let us know when to clean this up in the future.

Reply:
we could, however i am bit hesitant to bake in versions into source code. I think there is general consensus that devutils is a pain (in large part due to the versioning) and needs to be changed. 

given that this is git revert (old code (except the `file:///` part)) I would prefer to solve the larger issue of versioning more holistically.  

@andrewvc - were you going to log an issue based on our conversation for addressing devutils in the long term ?

Reply:
This seems to have stalled on a question to Andrew, but I think we can move forward the PR as is. Ry's point on future proofing is a valid one, but we definitely want to eliminate this gem, the annoyances it creates due to dependencies with core don't justify the benefit of isolating the small functionality it ships with.